pkg/metrics/metrics.go

```diff
@@ -416,6 +421,24 @@ var centralStatusSinceCreatedMetric = prometheus.NewGaugeVec(
 	centralStatusSinceCreatedMetricLabels,
 )
 
+var centralTimeoutCountMetric = prometheus.NewCounterVec(
+	prometheus.CounterOpts{
+		Subsystem: FleetManager,
+		Name:      "central_timeout_total_count",
```

Comment:
```suggestion
		Name:      "central_timeouts_total",
```
Prometheus counters end in `_total` by convention, the `count` is redundant.